src/libraries/HDGEOMETRY/DGeometry.cc

```diff
@@ -1771,7 +1771,8 @@ bool DGeometry::GetCCALZ(double &z_ccal) const
    bool good = Get("//section/composition/posXYZ[@volume='ComptonEMcal']/@X_Y_Z", ComptonEMcalpos);
 
    if(!good){
-      _DBG_<<"Unable to retrieve ComptonEMcal position."<<endl;
+	  // NEED TO RETHINK ERROR REPORTING FOR OPTIONAL DETECTOR ELEMENTS
```

Comment:
Is this "rethink" about the cases where a detector is not in the XML geometry and the Get() function reports a warning?  For example 
`src/JANA/JGeometryXML.cc:348 Node or attribute not found for xpath "//section/composition/posXYZ[@volume='ComptonEMcal']/@X_Y_Z".`

It's not for this PR, but should that be a separate issue?

Reply:
Right, the error messages are coming from the JANA level, so once we decide on a change on that level, we can revisit how to handle these cases.  We can open an issue on this or not - I don't have a strong opinion.

Enjoy the more compact command line output!